tests/integration/modifiers/autofocus-test.js

```diff
@@ -97,4 +97,15 @@ module('Integration | Modifier | autofocus', function(hooks) {
 
     assert.dom('[data-test-input-6]').isFocused('The custom selected element is focused');
   });
+
+  test('should give focus to the first included textarea that is not disabled', async function (assert) {
+    await render(hbs`
+      <form {{autofocus}}>
+        <textarea data-test-textarea="disabled" disabled />
+        <textarea data-test-textarea="enabled" />
```

Comment:
I would add an input to make sure that it really is the first input *or* textarea that get the input (in case the default selector get refactored). Otherwise, LGTM 👍 

```suggestion
        <textarea data-test-textarea="enabled" />
        <input data-test-input="enabled" />
```